denis/utilities.py

```diff
@@ -131,6 +131,42 @@ def check_signed_off_by(repo, tag):
     return msg
 
 
+def check_subject_tag(repo, tag):
+    [assignment, component, user] = tag.split('_')
+
+    msg = 'subject tag check'
+    msg += '\n'
+    msg += '-----------------'
+    msg += '\n\n'
+
+    commits = repo.git.execute(['git', 'rev-list', tag]).split('\n')
+
+    sub_tbl = mailman.db.Submission
+    relevant_submissions = (sub_tbl.select()
+                            .where(sub_tbl.recipient == assignment)
+                            .where(sub_tbl.user == user)
+                            .order_by(sub_tbl.timestamp.desc()))
+    expected_revision_number = relevant_submissions.count()
+
+    nr_commits = len(commits)
+    # from 0/n .. n/n
+    expected_max_index = str(nr_commits - 1)
```

Comment:
not sure if you need to `str` it since you are using it in an f-string below. You could even just drop the var alltogether and just inline `nr_commits - 1` into the `{}` of the fstring.

Reply:
inlined